src/pages/SinglePost/SinglePost.jsx

```diff
@@ -6,11 +6,7 @@ import Post from '../../components/SinglePost/SinglePost';
 const SinglePost = ({ match, location, history }) => (
   <Layout match={match} location={location} history={history}>
     <div className="container-fluid">
-      <div className="row">
-        <div className="col-md-8 offset-md-2">
-          <Post slug={match.params.slug} />
-        </div>
-      </div>
```

Comment:
@dusmel I do not understand why you deleted this!

Reply:
Because I couldn't add the new block in single post page, since the Post component only had `col-md-8`, I had to give it a full width then divide in the Post component.